### DISCUSSION TITLE
Recommendations for running beets on a media server

### DISCUSSION THREAD

**Discussion by thekakkun** [beetbox/beets#5090] [General]
Hi, I'm planning to run my own media server, for the first time, with plans to manage my music using beets.

Does anyone have any recommendations for how to set this up?
In particular, will I be accessing beets by ssh-ing into the machine and invoking the command line tool? Is there an easier way?

Thanks!

**Comment by jpecora716** [comment]
I run beets locally by mounting an nfs share from my NAS

**Reply by arsaboo** [reply]
Same, my music is on my Synology. I just mount that folder and access the files.

**Comment by Serene-Arc** [comment]
SSHing into the machine is what I do. It works perfectly well if you have remote terminal access, and is no different to running it locally. Additionally, there are docker containers for beets available that make it possible to have multiple libraries or configurations.
